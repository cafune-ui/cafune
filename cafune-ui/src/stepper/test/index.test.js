import Stepper from '../';
import { mount } from 'enzyme';
// import { mount, deep } from 'preact-render-spy';
describe('Stepper', () => {
  it('should render properly', () => {
    const wrapper = mount(<Stepper />);
    expect(wrapper).toMatchSnapshot();
  });
});
