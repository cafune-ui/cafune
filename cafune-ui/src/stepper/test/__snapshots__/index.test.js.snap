// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stepper should render properly 1`] = `ReactWrapper {}`;
